### PR TITLE
Document allow_conda_downgrades config param

### PR DESCRIPTION
**Pull request by nehaljwani**

### CODE REVIEW

File: conda/base/context.py [hunk 1]
```diff
@@ -592,7 +592,6 @@ def list_parameters(self):
             'bld_path',
             'conda_build',
             'croot',
-            'allow_conda_downgrades',
```
Comment (msarahan) [inline conda/base/context.py]: why is this removed?  I think it has to stay here?

Comment (nehaljwani) [reply conda/base/context.py]: This is the list of 'unlisted parameters'

Comment (msarahan) [reply conda/base/context.py]: Of course... Sorry.
